lib/justice_gov_sk/courts/resource_parser.rb

```diff
@@ -0,0 +1,78 @@
+module JusticeGovSk::Courts
+  class ResourceParser
+    def self.parse(html)
+      document = Nokogiri::HTML(html)
+      detail = document.css('.detail .right .inner > .content')
+      contact = detail.at_css('.contact-address')
+      information_center = detail.at_css('.informacne-centrum')
+      registry = detail.at_css('.podatelna')
+      business_registry = detail.at_css('.orsr')
+      coordinates = document.text.scan(/LatLng\("(\d{2}\.\d+)","(\d{2}.\d+)"\)/)[1]
+
+      HTMLCorrector.correct_contact_table(contact.css('table'))
+
+      attributes = {
+        nazov: detail.at_css('h1').text.strip.presence,
```

Comment:
Chcem som len pockat, aby som videl, co sa tam robi a ake hodnoty sa tam pridavaju, ale v konecnom dosledku ak by bol problem, tak by som to tam dal cez refinements - http://ruby-doc.org/core-2.1.1/doc/syntax/refinements_rdoc.html v ramci namespacu JusticeGovSk.